python/taichi/examples/rendering/cornell_box.py

```diff
@@ -1,512 +1,227 @@
-                nl = dot_or_zero(brdf_dir, hit_normal)
-                direct_li += fl * w * nl / brdf_pdf
-
-    return direct_li
-
-
-@ti.func
-def schlick(cos, eta):
-    r0 = (1.0 - eta) / (1.0 + eta)
-    r0 = r0 * r0
-    return r0 + (1 - r0) * ((1.0 - cos)**5)
-
-
-@ti.func
-def sample_ray_dir(indir, normal, hit_pos, mat):
-    u = ti.Vector([0.0, 0.0, 0.0])
-    pdf = 1.0
-    if mat == mat_lambertian:
-        u = sample_brdf(normal)
-        pdf = ti.max(eps, compute_brdf_pdf(normal, u))
-    elif mat == mat_specular:
-        u = reflect(indir, normal)
-    elif mat == mat_glass:
-        cos = indir.dot(normal)
-        ni_over_nt = refr_idx
-        outn = normal
-        if cos > 0.0:
-            outn = -normal
-            cos = refr_idx * cos
-        else:
-            ni_over_nt = 1.0 / refr_idx
-            cos = -cos
-        has_refr, refr_dir = refract(indir, outn, ni_over_nt)
-        refl_prob = 1.0
-        if has_refr:
-            refl_prob = schlick(cos, refr_idx)
-        if ti.random() < refl_prob:
-            u = reflect(indir, normal)
-        else:
-            u = refr_dir
-    return u.normalized(), pdf
-
-
-stratify_res = 5
-inv_stratify = 1.0 / 5.0
-
-
-@ti.kernel
-def render():
-    for u, v in color_buffer:
-        aspect_ratio = res[0] / res[1]
-        pos = camera_pos
-        cur_iter = count_var[0]
-        str_x, str_y = (cur_iter / stratify_res), (cur_iter % stratify_res)
-        ray_dir = ti.Vector([
-            (2 * fov * (u + (str_x + ti.random()) * inv_stratify) / res[1] -
-             fov * aspect_ratio - 1e-5),
-            (2 * fov * (v + (str_y + ti.random()) * inv_stratify) / res[1] -
-             fov - 1e-5),
-            -1.0,
-        ])
-        ray_dir = ray_dir.normalized()
-
-        acc_color = ti.Vector([0.0, 0.0, 0.0])
-        throughput = ti.Vector([1.0, 1.0, 1.0])
-
-        depth = 0
-        while depth < max_ray_depth:
-            closest, hit_normal, hit_color, mat = intersect_scene(pos, ray_dir)
-            if mat == mat_none:
-                break
-
-            hit_pos = pos + closest * ray_dir
-            hit_light = (mat == mat_light)
-            if hit_light:
-                acc_color += throughput * light_color
-                break
-            elif mat == mat_lambertian:
-                acc_color += throughput * sample_direct_light(
-                    hit_pos, hit_normal, hit_color)
-
-            depth += 1
-            ray_dir, pdf = sample_ray_dir(ray_dir, hit_normal, hit_pos, mat)
-            pos = hit_pos + 1e-4 * ray_dir
-            if mat == mat_lambertian:
-                throughput *= lambertian_brdf * hit_color * dot_or_zero(
-                    hit_normal, ray_dir) / pdf
-            else:
-                throughput *= hit_color
-        color_buffer[u, v] += acc_color
-    count_var[0] = (count_var[0] + 1) % (stratify_res * stratify_res)
+# pylint: disable=W0622,W0621,W0401
```

Comment:
This will not pass the pylint test. 

Use `# pylint: disable=W0622,W0621,W0401` or let me change the entire code to make it PEP8 compliant. 🥲 

```bash
pylint...................................................................Failed
- hook id: pylint
- exit code: 4

************* Module rendering.cornell_box
python\taichi\examples\rendering\cornell_box.py:3:0: W0622: Redefining built-in 'max' (redefined-builtin)
python\taichi\examples\rendering\cornell_box.py:3:0: W0622: Redefining built-in 'min' (redefined-builtin)
python\taichi\examples\rendering\cornell_box.py:3:0: W0622: Redefining built-in 'pow' (redefined-builtin)
python\taichi\examples\rendering\cornell_box.py:3:0: W0622: Redefining built-in 'round' (redefined-builtin)
python\taichi\examples\rendering\cornell_box.py:102:4: W0621: Redefining name 'e' from outer scope (line 3) (redefined-outer-name)
```

Reply:
Could you just use `disable=W0401` and see if it can pass the CI? I didn't encounter the problem you met when I remove `W0622` and `W0621`.  Sometimes the warning messages on different PC are different.

Reply:
Not in my case.